app/config/config.server.ts

```diff
@@ -8,14 +10,17 @@ interface Config {
 
 let instance: Config | undefined = undefined;
 
+const oidcClientSecretFilePath = "/etc/brak-idp-secrets/BRAK_IDP_OIDC_CLIENT_SECRET"
```

Comment:
I considered passing this hardcoded path as ENV var, but in my experience I've never changed this value, so I just hardcoded it. @manuelpuchta What do you think?